### ISSUE TITLE
word wrapping for long lines

### ISSUE THREAD

**Issue by rexim** [tsoding/cm#16]

**Comment by aodhneine** [comment]
Given that horizontal scrolling has been implemented as of #20, this issue should probably be closed.

**Comment by rexim** [comment]
@aodhneine not really. The idea was to make an option that enables word wrapping and disables horizontal scrolling (since they are mutually exclusive).
